app/Http/Controllers/WikiController.php

```diff
@@ -65,14 +65,25 @@ public function update($path)
     {
         priv_check('WikiPageRefresh')->ensureCan();
 
-        (new Wiki\Page($path, $this->locale()))->refresh();
+        (new Wiki\Page($path, $this->locale()))->forget();
 
         return ujs_redirect(Request::getUri());
     }
 
+    public function updateWebhook()
```

Comment:
Update the one in GithubImporter to handle this instead. And make sure of correct repository name, branch, event type, etc.

Reply:
Should this also use the currently existing webhook (ChangelogController@github) rather than this new one?

Reply:
yes